lib/main_common.pm

```diff
@@ -1100,8 +1100,6 @@ sub load_consoletests {
     loadtest "console/prepare_test_data";
     loadtest "console/consoletest_setup";
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
-    # This is a temporary solution for enabling open-vm-tools tests, will use YAML job template instead.
-    loadtest 'virt_autotest/esxi_open_vm_tools' if is_vmware && get_var('OPEN_VM_TOOLS');
```

Comment:
So you unplug open vm tools test completely from os-autoinst-distri-opensuse. The new textmode_svirt.yaml is the only place where open vm tools can be found.

Reply:
Yes, as the original design we should use yaml template for this new test instead of inserting test logic in main_common.pm. Besides, a part of console tests should not be included in textmode_svirt test suites.